### ISSUE TITLE
BUG: Redshift columns not showing up

### ISSUE THREAD

**Issue by not-night-but** [beekeeper-studio/beekeeper-studio#1632]
**Describe the bug**
From slack:
```
Hey, Been using BKS for a bit and my like…column preview functionality just doesn’t work anymore as of last week.  There is definitely data in the tables

I’m connecting to a Redshift db and using 3.9.17

I can never get autofill working either.
```

**Screenshots / Gifs**
https://beekeeperstud-lvg5276.slack.com/files/U05FX3ST1HD/F05GE43AD3N/image.png
https://beekeeperstud-lvg5276.slack.com/files/U05FX3ST1HD/F05G8SNVCKF/image.png

**Version Information (please complete the following information):**
 - OS: MacOS M1
 - App Version: 3.9.17
 - Database type and version: Redshift

**Comment by bcelenza** [comment]
I can echo that I've also seen this. In my case at least, the error in the console when attempting to view the columns of a redshift table:

```
error: function concat(character varying, "unknown", character varying, "unknown") does not exist
    at h.parseErrorMessage (app://./js/chunk-vendors.4014caf5.js:69:133889)
    at handlePacket (app://./js/chunk-vendors.4014caf5.js:69:130529)
    at h.parse (app://./js/chunk-vendors.4014caf5.js:69:129175)
    at TLSSocket.<anonymous> (app://./js/chunk-vendors.4014caf5.js:221:41380)
    at TLSSocket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:199:23)
```

**Comment by not-night-but** [comment]
@bcelenza ah okay. Well at least it seems to be repeatable then. I am just waiting on getting access to our redshift cluster so I can start testing this. Would you be able to write a small script to create a table that this happens on?

**Comment by DeadEye0112** [comment]
I confirm the problem, since july 2023, Beekeeper does not seem to work at all on Redshift databases

**Comment by choval** [comment]
Same here, happens on any table.
Fails with Redshift and Postgres connection type.

Last version columns were displayed correctly was v3.9.9

**Comment by not-night-but** [comment]
I believe this has been fixed and is in master, will be out with the next release

**Comment by rick-cti** [comment]
Hi! I'm running v4.1.11 but think I'm still seeing this bug; in a Redshift DB, if I right-click a table and choose "View data" it doesn't display anything (just a filter bar at the top). If I run a SELECT query via the query editor, it shows results in a table. Is this still pending a merge into a future release or am I perhaps seeing a different issue? Thanks! 🙏
